docs/conf/australia/2022/welcome-wagon.rst

```diff
@@ -148,39 +148,31 @@ Join an unconference session
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 {% if flaghasschedule %}
-- Starting Monday morning, check the `Unconference schedule <{{unconf.url}}>`__ to see if there are any sessions you are interested in joining. New ones are added all the time, so check back periodically.
+- Starting Thursday morning, check the `Unconference schedule <{{unconf.url}}>`__ to see if there are any sessions you are interested in joining. New ones are added all the time, so check back periodically.
 - At the time the session starts, go to **Sessions** and select the session with the table number that corresponds to the one in the `Unconference schedule <{{unconf.url}}>`__.
 {% else %}
-- Starting Monday morning, check the Unconference schedule to see if there are any sessions you are interested in joining. New ones are added all the time, so check back periodically.
+- Starting Friday morning, check the Unconference schedule to see if there are any sessions you are interested in joining. New ones are added all the time, so check back periodically.
```

Comment:
This is the same text as L151, just here in the version without the link. It's only rendered before the schedule is published, just fixing for consistency.
```suggestion
- Starting Thursday morning, check the Unconference schedule to see if there are any sessions you are interested in joining. New ones are added all the time, so check back periodically.
```